src/plugin/plugin.ts

```diff
@@ -667,19 +673,20 @@ class Purchases {
    * Sets up Purchases with your API key and an app user id.
    * @param {PurchasesConfiguration} Object containing configuration parameters
    */
-   public static configureWith({
+  public static configureWith({
     apiKey,
     appUserID = null,
     observerMode = false,
     userDefaultsSuiteName,
+    usesStoreKit2IfAvailable = false,
```

Comment:
I think this doesn't break compilation for anyone because PurchasesConfiguration is an object and we are basically just adding a property with a default, so it doesn't matter we don't add it at the end of the object.

Reply:
I didn't add it to the other configure function because that will actually break